docs/marketplace-required-csv-annotations.md

```diff
@@ -0,0 +1,85 @@
+# Marketplace Required CSV Annotations
+
+An operator's CSV must contain the following annotations for it to be displayed properly within the Marketplace UI:
+
+```yaml
+metadata:
+  annotations:
+    categories: A list of comma separated categories that your operator falls under.
+    certified: The operator's certification. A process to get this value will be defined at a later date.
+    description: |-
+      A short description of the operator that will be displayed on the marketplace tile.
+      If this annotation is not present, the `spec.description` value will be shown instead.
+      In either case, only the first 135 characters will appear.
+    containerImage: The repository that hosts the operator image.
+    createdAt: The date that the operator was created.
```

Comment:
Need to tell people the format. Standard kube format is `2019-01-03T16:29:24Z`

Reply:
This should be the date the CSV was created in Quay. This assumes we don't touch CSVs in Quay if they don't need to be updated.

Reply:
One note, this value won't be identical to the date/time that the quay repo was updated because it has to be set within the csv, and users (or the system that automates pushing the csv) would have to update it before the push @robszumski

Reply:
that is ok with me